src/api/.keystone/config.js

```diff
@@ -796,6 +830,32 @@ var generate = {
 };
 var generate_otp_default = generate;
 
+// helpers/generate-otp-and-update-account.ts
+var generateOTPAndUpdateAccount = async (context, accountId) => {
+  try {
+    console.info("Adding OTP to exporter account");
```

Comment:
Do we need `accountId` validation or is it already validated?

Reply:
`accountId` and checking that the account exists is already in place before this function is called e.g

```js
const exporter = await getAccountByField(context, 'email', email);

if (!exporter) {
  console.info('Unable to validate exporter account - no account found');

  return { success: false };
}
```